docs/source/traits_user_manual/defining.rst

```diff
@@ -408,7 +408,7 @@ the table.
 | Union            | Union( *val1*\ [, *val2*, ..., *valN*,                   |
 |                  | \*\*\ *metadata*] )                                      |
 +------------------+----------------------------------------------------------+
-| UUID [4]_        | UUID( [\*\*\ *metadata*] )                               |
+| UUID             | UUID( [\*\*\ *metadata*] )                               |
```

Comment:
Dropped the `[4]` footnote, which talked about Python 2.5; I've re-used it for `Either`.

Reply:
Nitpick: footnote 4 comes before footnote 3 (attached to `Function` and `Method`) in the text.

Reply:
Thanks; fixed. At the same time, I fixed that the current footnotes start from the third natural number, rather than the second as is more conventional.